### DISCUSSION TITLE
why the vs code snippets are not working

### DISCUSSION THREAD

**Discussion by sahil-1012** [microsoft/vscode-discussions#439] [Extension Development QnA] [answered]
Can u pls check after the last update of VS CODE, my user defined snippets are not working in C, C++, and Python… I defined a lot of snippets & when I press my triggering word let's suppose I had defined [1]“py_main” as my triggering word for my snippets…

So, if I type “p” it shows the snippet in the suggestion tab, but when I start writing “py” the snippet suggestion from the menu disappears… If we continue adding one letter after other after reaching “py_m” it again starts appearing in the suggestion menu…

This is just an example, if I go for let's say [3]“cout” (as my user defined snippet) it shows me snippet suggestion till c but then as I type "co"[2]to make it come up, but instead of coming up it gets disappears and I'm only able to see the variable or keywords suggestions...

It's so annoying pls guys try to fix it this is all happening in a day or two… and it makes the edge of all over VS code over other editor to zero…

Another key observation I observed, is when I write “.c” (for my cout snippet) and then start typing surprisingly this time the snippet is shown in the menu...
Also, if I write "int" before my snippet triggering words the snippet doesn't disappear.

This issue is not happening in Java, Js...

[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/hrWR0.png
  [2]: https://i.stack.imgur.com/2hahU.png
  [3]: https://i.stack.imgur.com/clO4n.png

**Comment by gjsjohnmurray** [comment, marked as answer]
This place is currently focusing on helping people who are developing VS Code extensions. For issues like yours please visit the main repository. For example, see https://github.com/microsoft/vscode/issues/173416

**Comment by gjsjohnmurray** [comment]
More specifically, see issue 173244 there, and watch for a fix probably coming in 1.75.1
